.gitignore

```diff
@@ -0,0 +1,24 @@
+*.log
```

Comment:
This file is added here and in #5 as well. I'd add it on a separate commit or PR actually.